src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -681,9 +681,14 @@ public override bool HandleCall (MethodBody callingMethodBody, MethodReference c
 							if (value is SystemTypeValue typeValue) {
 								foreach (var genericParameter in typeValue.TypeRepresented.GenericParameters) {
 									if (_context.Annotations.FlowAnnotations.GetGenericParameterAnnotation (genericParameter) != DynamicallyAccessedMemberTypes.None ||
-										genericParameter.HasDefaultConstructorConstraint) {
+										(genericParameter.HasDefaultConstructorConstraint && !typeValue.TypeRepresented.IsTypeOf ("System", "Nullable"))) {
```

Comment:
I don't remember - does Cecil also try to unmangle C# or should this rather be

```suggestion
										(genericParameter.HasDefaultConstructorConstraint && !typeValue.TypeRepresented.IsTypeOf ("System", "Nullable`1"))) {
```

Reply:
You're right - I wrote a test for it, but then forgot to call the test :-(
All fixed now.